### ISSUE TITLE
Craft 5 migration fails because of Formie

### ISSUE THREAD

**Issue by sarahschuetz** [verbb/formie#1979]
### Describe the bug

When updating a Project to Craft 5.2.6 (and Formie to version 3.0.0-beta.16 at the same time), one of Craft's migration fails with the following error:

```
*** applying m230511_000000_field_layout_configs
Exception: Formie fields’ `name` attribute has been deprecated. Use `label` instead. (/Users/sarah/.../vendor/yiisoft/yii2/base/Component.php:180)
#0 /Users/sarah/.../vendor/verbb/formie/src/base/Field.php(1725): craft\services\Deprecator->log('verbb\\formie\\ba...', 'Formie fields\xE2\x80...')
...
```

Any ideas on what the problem might be? I am not sure, how to fix this since the deprecation error is not in my custom code but one of Craft's own migrations.

### Steps to reproduce

1. Install Craft 5.2.6 and Formie 3.0.0-beta.16
2. Run `php craft up`

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Not sure. Is this relevant for migration errors? If yes, let me know and I will check!

### Craft CMS version

5.2.6

### Plugin version

3.0.0-beta.16

### Multi-site?

yes

**Comment by engram-design** [comment]
These deprecation **warnings** are normal during the upgrade process of Formie 2 to 3, and by default they are warnings. Once migrated, they are no longer triggered, this is because fields are bootstrapped so early in the process with Craft that these warnings will appear almost immediately before an upgrade can take place.

However, it shouldn't be an exception that stops the migration altogether. This is likely due to setting deprecation warnings to be shown as fatal errors as per [this official guide](https://craftcms.com/knowledge-base/preparing-for-craft-4#fix-deprecation-warnings). While that's useful in most cases, that's going to cause issues when Formie is trying to run its migrations.

The best course of action is to disable that exception-throwing handling while Formie does its migrations.

**Comment by sarahschuetz** [comment]
That makes total sense, I didn't think of that. Thank you! 🚀

**Comment by jamesmacwhite** [comment]
I'm getting this on a site with Formie 3 installed when going to Add Exising Field on a form `/actions/formie/forms/get-existing-fields`

Because of having throw exceptions on for the deprecator is causes a 500.

If you pass the YII_DEBUG constant as the value, you can control the state through CRAFT_DEV_MODE being on/off, but not sure why this is being thrown when already installed.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.9"`.

**Comment by jamesmacwhite** [comment]
Ah, that would explain it. Thanks @engram-design!

**Comment by engram-design** [comment]
Fixed in [3.0.10](https://github.com/verbb/formie/releases/tag/3.0.10)
